### PR TITLE
Updated docker file

### PR DESCRIPTION
**Pull request by srossross**
@PeterDSteinberg  can you review?

### CODE REVIEW

File: docker/linux-64/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,25 @@
 
 FROM binstar/build-base:centos6
 
+
+RUN yum install -y gcc gcc-c++
```
Comment (srossross) [inline docker/linux-64/Dockerfile]: Not sure why, but conda gcc does not seem to work anymore.

File: docker/linux-64/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+ENV HOME /home/anaconda
 ENV PATH /opt/miniconda/bin:$PATH
 
+
+
 RUN conda config --set always_yes true && \
     conda config --set binstar_upload false && \
     conda config --add channels binstar
-RUN conda install conda-build jinja2 binstar psutil binstar-build llvm gcc
-RUN binstar config --set sites.binstar.url "https://api.anaconda.org"
```
Comment (goanpeca) [inline docker/linux-64/Dockerfile]: Why is this being removed?
`RUN binstar config --set sites.binstar.url "https://api.anaconda.org"`

Comment (srossross) [inline docker/linux-64/Dockerfile]: Because that is the default in the latest releases.

**Comment by srossross** [comment]
Any objections to merging this?

**Pull request merged**